jbrowse/src/client/JBrowse/Browser/plugins/ExtendedVariantPlugin/ExtendedVariantRenderer/components/Segments.js

```diff
@@ -0,0 +1,104 @@
+import { readConfObject } from '@jbrowse/core/configuration'
```

Comment:
If any file is 100% cut/paste from JB2, can you add a comment near the top explicitly saying that? it will make it clearer what can/should go when we clean up this part of the code

Reply:
Updated